ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml

```diff
@@ -59,10 +59,18 @@ chain:
 
       # Data for cluster bot.
       # The kubeadmin-password secret is reconciled only after the kas is available so giving a few seconds to avoid racing and move forward otherwise. 
-      sleep 15
-      echo "Getting ${CLUSTER_NAME}-kubeadmin-password from namespace ${HYPERSHIFT_NAMESPACE}"
-      oc get secret -n${HYPERSHIFT_NAMESPACE} ${CLUSTER_NAME}-kubeadmin-password --template={{.data.password}} | base64 -d > ${SHARED_DIR}/kubeadmin-password
-
+      timeout --foreground 5m bash <<- "EOF"
```

Comment:
if we happen to have a bug on this logic or in the core logic that creates the secret we don't want to block jobs for 5m here.

This was only used so cluster-bot could print the value.